src/OpenProject/WebViewIntegration/LandingPage/index.html

```diff
@@ -351,10 +326,6 @@
           window.RevitBridge.sendMessageToOpenProject = function (messageData) {
             const message = JSON.parse(messageData);
 
-            if (message.messagePayload && message.messagePayload.length) {
-              message.messagePayload = JSON.parse(message.messagePayload);
```

Comment:
The payload itself is a string, are you sure we don’t need this?

Reply:
I thought you were stringifying the entire object. If messagePayload is an stringified object itself, then we need to keep this code. I'll add it again, thanks.